view/widgets/acquisition_widgets/volume_widget.py

```diff
@@ -451,6 +453,22 @@ def toggle_z_show(self, current_row, current_column, previous_row, previous_colu
             z = self.scan_plan_widget.z_plan_widgets[show_row, show_col]
             self.scan_plan_widget.stacked_widget.setCurrentWidget(z)
 
+    def handle_snapshot(self, image):
```

Comment:
Since this isn't a napari viewer, I am guessing multi-resolution images aren't supported. So I think this will freakout in the GL window if the massive exaspim images are not downsampled first. This tile navigator isn't zoomable so I think it is fine to downsample images into the GL window.

We need an internal criteria of how to do this. One fancy option would be to figure out how many pixels big the GL viewer is and then based on that, how big a tile is in pixels, and the subsample the snapshot to be approximately that size. This might be overkill, another option is to do something simpler and say any snapshot should be say ~1 MB which at uint16 would be ~1000x500 pixels. Downsampling the input image by 2^N in XY until it falls around or below 1 MB in size.